graylog2-server/src/main/java/org/graylog/plugins/pipelineprocessor/functions/conversion/IsMap.java

```diff
@@ -48,6 +49,10 @@ public FunctionDescriptor<Boolean> descriptor() {
                 .returnType(Boolean.class)
                 .params(valueParam)
                 .description("Checks whether a value is a map")
+                .ruleBuilderEnabled(false)
+                .ruleBuilderName("Check if map")
+                .ruleBuilderTitle("Check if '${value}' is a map")
```

Comment:
[Grammar](https://www.merriam-webster.com/grammar/if-vs-whether-difference-usage#:~:text=For%20clarity%2C%20it%20is%20best,clauses%2C%20phrases%2C%20or%20words.) OCD comment: _For clarity, it is best to use whether in reference to a choice or alternatives ("we're going whether it rains or not") and if when establishing a condition ("we will go if it doesn't rain")._

Using _if_ implies the meaning: If value is map perform some unspecified check. _Whether_ expresses the intended meaning: Determine whether value is map or not map.

I'm fine keeping _if_ here - it's not really ambiguous and it's shorter.

Reply:
Keeping it for the name, changing it for the title